src/libstd/keyword_docs.rs

```diff
@@ -29,9 +29,14 @@
 /// [`crate`]: keyword.crate.html
 mod as_keyword { }
 
+#[doc(keyword = "break")]
+//
+/// Exit early from a loop.
+mod break_keyword { }
+
 #[doc(keyword = "const")]
 //
-/// The keyword for defining constants.
+/// Compute something at compile-time.
```

Comment:
Not true since `const fn` got stable.

Reply:
I considered "Assert that something can be computed at compile-time" which applies to both `const` declarations and `const fn`s and is technically more correct but also more verbose.

Reply:
This is not the full story of `const fn` and is in my view quite misleading. A `const fn` is a deterministic function that *can* be executed at CTFE if given CT-evaluable arguments. A `const fn` can also be evaluated at runtime.

Reply:
It's probably not a good idea to put the full story into a single-sentence title.
"Compute something at compile-time" seems both close and vague enough.

Reply:
If you want to be vague, I'd at least throw in a "can" here since a `const fn` need not be evaluated at CT.

Reply:
My preference would be for something like:

```suggestion
/// Compile time constants and deterministic functions.
```

I think conflating `const fn` with `const` spreads a misconception I'm not too happy about.